### PR TITLE
Merge-able version of 1.0.0

### PR DESCRIPTION
**Pull request by jordy25519**
Same as #179 with merged `develop`.
Don't want to change #179 so it's easier to do this rather than managing CI to stop the deployment of a new image

Changes:
- Adds MainNet genesis config
- Removes Rimu genesis config

### CODE REVIEW

**Review by aliXsed — APPROVED** [review]

File: crml/cennzx-spot/src/tests.rs [hunk 1]
```diff
@@ -23,9 +23,201 @@ use crate::{
 	CoreAssetId, Error, Trait,
 };
 use core::convert::TryFrom;
-use frame_support::{traits::Currency, StorageValue};
-use mock::{AccountId, CennzXSpot, ExtBuilder, Origin, Test};
-use sp_core::{crypto::UncheckedInto, H256};
+use frame_support::{additional_traits::DummyDispatchVerifier, impl_outer_origin, traits::Currency, StorageValue};
```
Comment (jordy25519) [inline crml/cennzx-spot/src/tests.rs]: will rebase

Comment (hoani) [inline crml/cennzx-spot/src/tests.rs]: This shouldn't be here - It was meant to be pulled out and put into the mock. Hopefully a rebase fixes this?

**Review by cowboy-bebug — COMMENTED** [review]
LGTM! Can be merged after the `cennzx-spot/tests.rs` is rebased

**Review by hoani — APPROVED** [review]
LGTM
Looks like there might be some weirdness with CENNZxSpot, should be easy to rebase and sort out.

**Review by roy-powerfinance — APPROVED** [review]

**Pull request merged**